.github/workflows/lint-test.yml

```diff
@@ -0,0 +1,91 @@
+name: Lint & Test
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  push:
+    branches:
+      - '**'
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
AFAIK v2 is deprecated and v3 is used below, is this copy-pasta, or intentional?

Reply:
yep this was unintentional